### DISCUSSION TITLE
Core 2 Duo support

### DISCUSSION THREAD

**Discussion by bobtus** [sebanc/brunch#1415] [Q&A]
Hello will support for Intel Core 2 Duo CPUs be added? Thanks!

**Comment by tejasraman** [comment]
FYI ChromeOS itself never had support for Core 2 Duo. AFAIK there are no plans to support it though. You could try CloudReady though.

**Comment by bobtus** [comment]
Yes I know. I tried CloudReady already, but did not like it. That is why I was wondering about brunch to support it

**Comment by tejasraman** [comment]
Told you, C2D was never supported in official ChromeOS. Only 2nd Gen Intel and newer is supported
